src/util/simplify_expr_class.h

```diff
@@ -15,6 +15,8 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <sys/stat.h>
 #endif
 
+#include <expr.h>
+#include <std_expr.h>
```

Comment:
These should be `util/*` each.

Also, I notice these weren't previous included for some reason, does anyone know why people opted to forward declare everything before?